airbyte-cdk/python/airbyte_cdk/sources/declarative/models/declarative_component_schema.py

```diff
@@ -180,6 +180,26 @@ class OAuthAuthenticator(BaseModel):
     parameters: Optional[Dict[str, Any]] = Field(None, alias="$parameters")
 
 
+class SingleUseRefreshTokenOAuthAuthenticator(BaseModel):
+    type: Literal["SingleUseRefreshTokenOAuthAuthenticator"]
+    token_refresh_endpoint: str
+    client_id_config_path: Optional[List[str]] = ["credentials", "client_id"]
```

Comment:
While this is inconsistent with the other authenticators, I think this is a good way to avoid the need for string interpolation since we know the client_id should come from the config. 

CC @brianjlai - I'm not particularly interested in introducing more widespread breaking changes, but I think this might be a good pattern to use when possible